### DISCUSSION TITLE
Future of vscode and Bob integration

### DISCUSSION THREAD

**Discussion by pabloto** [codefori/vscode-ibmi#747] [General]
Hi,

has you have announced now you are part of IBM, so what's can we expect from this wonderful project? 

Until now my colleague esitate to use VsCode and still continue to use SEU because your extension doesn't have any IBM certification, Could it be possibile the Code for I will be part of IBM?

About Bob integration, How do I find some example?

Many thanks to you and to all Code for I contributors.

**Comment by worksofliam** [comment]
Hi @pabloto!

The project will continue on as normal right now. Nothing special lined up other than the planned features - this includes further bob support.

Code for IBM i has great support for local workspaces, and I recommend you check out this video I did which shows developing locally (but you can replace make with bob): https://www.youtube.com/watch?v=XuiGyWptgDA&t=356s

**Reply by pabloto** [reply]
Hi @worksofliam ,

I work in local and deploy to IBM i, using action.json and makefile, but It was not clear for me the integration with bob.

Do you have same example?

**Reply by worksofliam** [reply]
@pabloto Bob would just run in place of make. Check out the docs here for setting up your `actions.json` with `makei`.

https://ibm.github.io/ibmi-bob/#/integration/code-for-ibm-i?id=developing-on-the-local-machine-with-ibmi-bob

**Reply by edmundreinhardt** [reply]
Very cool documentation!

**Comment by edmundreinhardt** [comment]
While this project is not "IBM certified" it is open source that is being maintained by IBMers.  I should hope that is sufficient endorsement to use it instead of SEU, which by the way is NOT IBM enhanced any more.
